api/common.go

```diff
@@ -15,11 +15,11 @@ import (
 	"github.com/docker/libtrust"
 )
 
+// Version of Current REST API
+var Version = version.Version("1.22")
```

Comment:
If we did that I'd like to do it for MinVersion and DefaultDockerfileName too.
Didn't see any common object that is shared between the cli and server - like a context object (server has one, sometimes, but cli doesn't).  Not sure its cleaner to move this into two spots at this time and run the risk of them getting out of sync or confusing people.